env0/resource_project_policy.go

```diff
@@ -111,6 +111,12 @@ func resourceProjectPolicy() *schema.Resource {
 				Optional:         true,
 				ValidateDiagFunc: ValidateCronExpression,
 			},
+			"vcs_pr_comments_enabled_default": {
+				Type:        schema.TypeBool,
+				Description: "if 'true' all environments created in this project via the UI will be created with an 'enabled' running VCS PR commands using PR comments. Note: in the provider when creating an environment this is ignored, and must be configured explicitly by setting 'vcs_pr_comments_enabled' to 'true' or setting an alias in 'vcs_commands_alias'.",
```

Comment:
See the note.
This is a design decision for the provider.

Reply:
nice. But what do you mean by `design decision for the provider.`?
This is the purpose of the project policies, no?

Reply:
going to make changes based on our discussion.

Reply:
@TomerHeber Not sure I got this too.
it should be something like this:
```suggestion
				Description: "if 'true' all new environments created in this project will be created with an 'enabled' VCS PR commands using PR comments.",
```

Reply:
for a user to enable this feature, he should either provide 
'vcs_pr_comments_enabled' to 'true' or setting an alias in 'vcs_commands_alias', or set both which is explicit.

we can add a warning log when the user uses only `vcs_commands_alias`, this is just a suggestion. 
Anyways the BE takes care of it and forcibly set `vcs_pr_comments_enabled` if needed (not depending on client UI/provider/API)

Reply:
let me know if we didnt understand you

Reply:
Indeed if the user enabled the project policy - but didn't add `vcs_pr_comments_enabled` on the environment resource IaC. this is a problem.

how we should handle this?